src/desktop/components/artwork_filter_2/models/filter.coffee

```diff
@@ -38,7 +38,6 @@ module.exports = class ArtworkFilter extends Backbone.Model
       _.extend(props, { @page })
 
     variables = _.extend(
-      @params.defaultParams,
```

Comment:
This was what was causing all the misery over the past few prs. Even though i don't see any place where this is being directly mutated, it was being mutated! Removing it seemed to fix a whole bunch of things including that `@set param, null` -> `@unset param` thing from the other PR